src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -571,10 +571,15 @@ private StudentEnrollDetails enrollStudent(StudentAttributes validStudentAttribu
         return enrollmentDetails;
     }
 
-    /* All empty lines or lines with only white spaces will be skipped.
-     * The invalidity info is in HTML format.
+    /**
+     * This method builds the studentList from user input lines and check the validity of each student instance created.
```

Comment:
` and check the validity of each student instance created` is not needed because it is implied by the `@throws` clause description.